cmd/mtc/main.go

```diff
@@ -339,14 +343,15 @@ func assertionFromFlagsUnchecked(cc *cli.Context) (*ca.QueuedAssertion, error) {
 		a.Subject = subj
 	}
 
-	return &ca.QueuedAssertion{
+	return &mtc.AssertionRequest{
 		Assertion: a,
+		Evidence:  e,
 		Checksum:  checksum,
 	}, nil
 }
 
 func handleCaQueue(cc *cli.Context) error {
-	qa, err := assertionFromFlags(cc)
+	a, err := assertionFromFlags(cc)
```

Comment:
Note: Renaming `qa` to `a` everywhere makes me worried there might be a shadow of an existing `a` variable. Haven't seen one so far.

Reply:
Good point--I'll switch to `ar` everywhere instead.